pkg/providers/property/data_akamai_search_test.go

```diff
@@ -0,0 +1,70 @@
+package property
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/akamai/AkamaiOPEN-edgegrid-golang/v2/pkg/papi"
+)
+
+func TestDSSearch(t *testing.T) {
+	t.Run("match by hostname", func(t *testing.T) {
```

Comment:
Could you please add some error test? For example: what if `client.SearchProperties()` returns an error?

Reply:
added just one

Reply:
That's fine. If you do not mind, we can also add it ourselves to this PR. Up to you :-)

Reply:
Feel free to edit this PR as you wish!